ern-local-cli/src/commands/create-api-impl.js

```diff
@@ -64,10 +64,12 @@ exports.handler = async function ({
   outputDirectory: string,
   hasConfig: boolean
 }) {
-  const isPackageNameInNpm = await utils.doesPackageExistInNpm(api)
+  // check if the packageName for specified {apiName}-impl exists
+  const apiImpl = api.concat('-impl')
```

Comment:
use ES6 template instead? `${api}-impl` ?

Reply:
Makes sense to use string interpolation